### PR TITLE
Development threebotdeployerwallet

### PR DESCRIPTION
**Pull request by waleedhammam**
### Description

- 3Bot deployer to start with a funded wallet
- when the user wants to create an instance, we create a pool for this instance using the funded wallet for 15 mins (that should be enough for the initialization step).
- When we manage to deploy and initialize the 3Bot, we ask the user to extend the lifetime of the 3Bot in the same chatflow
- In case of pool extension failure they will be refunded from the explorer

### Related Issues

- https://github.com/threefoldtech/js-sdk/issues/1167

### CODE REVIEW

File: jumpscale/packages/threebot_deployer/chats/threebot.py [hunk 1]
```diff
@@ -319,5 +237,34 @@ def initializing(self):
...
+        self.pool = j.sals.zos.pools.get(self.pool_id)
+        if self.pool.empty_at < DURATION_MAX:
+            # Pool currently being consumed (compute or storage), default is current pool empty at + 65 mins
+            min_timestamp_fromnow = self.pool.empty_at - j.data.time.get().timestamp
+            default_time = self.pool.empty_at + 3900
+        else:
+            # Pool not being consumed (compute or storage), default is in 14 days (60*60*24*14 = 1209600)
+            min_timestamp_fromnow = None
+            default_time = j.data.time.get().timestamp + 1209600
+        msg = """Please enter the expiration date of your 3Bot. This will be used to calculate the amount of capacity you need to keep your 3Bot alive and build projects on top of the TF Grid. But no worries, you could always extend your 3Bot’s lifetime on 3Bot Deployer's home screen"""
```
Comment (rkhamis) [inline jumpscale/packages/threebot_deployer/chats/threebot.py:252]: but no worries,... extend your 3Bot's lifetime on the 3Bot Deployer's home screen

**Review by rkhamis — CHANGES_REQUESTED** [review]

**Review by ranatrk — APPROVED** [review]

File: jumpscale/packages/threebot_deployer/chats/threebot.py (hunk 1 above)
Comment (waleedhammam) [reply jumpscale/packages/threebot_deployer/chats/threebot.py]: added, thanks

File: jumpscale/sals/marketplace/deployer.py [hunk 2]
```diff
@@ -68,9 +68,12 @@ def show_payment(self, pool, bot):
         bot.qrcode_show(data=qr_code, msg=msg_text, scale=4, update=True, html=True)
         return qr_code
 
-    def pay_for_pool(self, pool):
+    def pay_for_pool(self, pool, is_threebot=False):
```
Comment (xmonader) [inline jumpscale/sals/marketplace/deployer.py:71]: let's unify the wallet name

Comment (waleedhammam) [reply jumpscale/sals/marketplace/deployer.py]: done, thanks

**Review by xmonader — APPROVED** [review]

**Pull request merged**